spec/services/idv/profile_maker_spec.rb

```diff
@@ -76,17 +81,21 @@
         expect(profile.initiating_service_provider).to eq(nil)
         expect(profile.verified_at).to be_nil
       end
+      it 'marks the profile as legacy_unsupervised' do
+        expect(profile.idv_level).to eql('legacy_unsupervised')
+      end
     end
 
     context 'with gpo_verification_needed' do
-      it 'deactivates a profile for gpo verification' do
-        profile = subject.save_profile(
+      let(:profile) do
+        subject.save_profile(
           fraud_pending_reason: nil,
           gpo_verification_needed: true,
           deactivation_reason: nil,
           in_person_verification_needed: false,
         )
-
+      end
+      it 'deactivates a profile for gpo verification' do
```

Comment:
We're not deactivating profiles for gpo verification anymore, we create them as pending
```suggestion
      it 'creates a pending profile for gpo verification' do
```